homeassistant/components/wemo/__init__.py

```diff
@@ -183,12 +178,15 @@ class WemoDiscovery:
     ADDITIONAL_SECONDS_BETWEEN_SCANS = 10
     MAX_SECONDS_BETWEEN_SCANS = 300
 
-    def __init__(self, hass: HomeAssistant, wemo_dispatcher: WemoDispatcher) -> None:
+    def __init__(
+        self, hass: HomeAssistant, wemo_dispatcher: WemoDispatcher, staticConfig
```

Comment:
Please use snake_case names for parameter names.

Also please type the signature completely.